src/llvm/lib_llvm.cr

```diff
@@ -33,16 +33,18 @@ end
     IS_40 = {{LibLLVM::VERSION.starts_with?("4.0")}}
     IS_39 = {{LibLLVM::VERSION.starts_with?("3.9")}}
     IS_38 = {{LibLLVM::VERSION.starts_with?("3.8")}}
+  {% end %}
 
-    IS_LT_70 = IS_38 || IS_39 || IS_40 || IS_50 || IS_60
-    IS_LT_80 = IS_LT_70 || IS_70 || IS_71
-    IS_LT_90 = IS_LT_80 || IS_80
-    IS_LT_100 = IS_LT_90 || IS_90
-    IS_LT_110 = IS_LT_100 || IS_100
-    IS_LT_120 = IS_LT_110 || IS_110 || IS_111
-    IS_LT_130 = IS_LT_120 || IS_120
-  end
-{% end %}
+  {% begin %}
```

Comment:
I think I'm missing something here. I assume in the original code the lack of use of `compare_versions` is for performance. Is that so? Do we have numbers to back that up? I mean, it'd be much cleaner to write `{{ compare_versions(LibLLVM::VERSION, "13.0.0" }}'

Reply:
AFAICT LLVM uses x.x.x format since 3.4.1, so I would assume this should work (ref: https://releases.llvm.org/).